ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-presubmits.yaml

```diff
@@ -6331,7 +6331,7 @@ presubmits:
     - ^main$
     - ^main-
     cluster: build05
-    context: ci/prow/azure-4.15-nightly-x86-control-plane-ipsec-24nodes
+    context: ci/prow/azure-4.15-nightly-x86-control-plane-ipsec-3nodes
```

Comment:
i'm not following where "3" came from here? Looks like all you are changing in the config is 24 down to 8 nodes right?

Reply:
There is already an existing job for 3 nodes, hence job generation swapped the jobs as per ascending order.